cluster/saltbase/salt/fluentd-es/fluentd-es.yaml

```diff
@@ -5,20 +5,20 @@ metadata:
 spec:
   containers:
   - name: fluentd-elasticsearch
-    image: gcr.io/google_containers/fluentd-elasticsearch:1.5
-    env:
-    - name: "FLUENTD_ARGS"
-      value: "-qq"
+    image: gcr.io/google_containers/fluentd-elasticsearch:1.6
+    args:
+    - -qq
     volumeMounts:
     - name: varlog
-      mountPath: /varlog
-    - name: containers
+      mountPath: /var/log
```

Comment:
It's fairly standard practice to shadow unused directories in the container with the corresponding host dirs, lots of examples all over the place. It feels more natural to run a containerized process as similar to running it straight on the host as possible, but it's no biggie to change if need be.